docs/website/content/2.configuration/4.block/rate_limit.md

```diff
@@ -10,10 +10,9 @@ Rate limiting protects backend services. It implements quota management used to
 | :-------------- | :-------------------- | :--------------- | :--------------- | :--------------- | :--------------- |
 | `period`        | [duration](#duration) | - | Defines the rate limit period. | &#9888; required, must not be negative or `0` (zero) | `period = "1m"` |
 | `per_period`    | integer               | - | Defines the number of allowed backend requests in a period. | &#9888; required, must not be negative or `0` (zero) | `per_period = 100` |
-| `period_window` | string                | `"sliding"` | Defines the window of the period. A `fixed` window always starts with the first request to the parent `backend`. A `sliding` period is a period from the time of the client request that lies behind, e.g. `10:23:44-11:23:45 a.m.` if the request time is `11:23:45 a.m.` and `period = "1h"` is defined. | Allowed values: `"fixed"` or `"sliding"` | `period_window = "sliding"` |
+| `period_window` | string                | `"sliding"` | Defines the window of the period. A `fixed` window permits `per_period` requests within `period` after the first request to the parent backend. After the `period` has expired, another `per_period` requests are permitted. The `sliding` window ensures that only `per_period` requests are sent in any interval of length `period`. | Allowed values: `"fixed"` or `"sliding"` | `period_window = "sliding"` |
+| `mode`          | string                | `"wait"` | If `mode` is set to `block` and the rate limit is exceeded, the client request is immediately answered with HTTP status code `429 Too Many Requests` and no backend request is made. If `mode` is set to `wait` and the rate limit is exceeded, the request waits for the next free rate limiting period. | Allowed values: `"wait"` or `"block"` | `mode = "wait"` |
```

Comment:
Nitpick: the status _code_ is the 3-digit integer; together with the reason phrase it's the status _line_. So: "answered with HTTP status line ..." or just "answered with HTTP status ...".

Reply:
Done